style/rails/README.md

```diff
@@ -14,7 +14,8 @@ Rails
 * Order ActiveRecord associations above ActiveRecord validations.
 * Order controller contents: filters, public methods, private methods.
 * Order i18n translations alphabetically by key name.
-* Order model contents: constants, macros, public methods, private methods.
+* Order model contents: constants, modules, macros, public methods,
```

Comment:
It's clear what "modules" means here with the example, but I don't think it's clear without it. What about "included modules" or "included and extended modules"?